### ISSUE TITLE
VGMtrans not writing SF2 from .bd/.hd files

### ISSUE THREAD

**Issue by nero1024** [vgmtrans/vgmtrans#111]
I cannot seem to be able convince VGMtrans to write a .sf2 SoundFont from a standard PS2 soundbank (.bd, .bh). No SoundFont file is written after selecting the detected PS2 InstrSet and using the "Convert to SoundFont 2" item from the right-click context menu. No error was reported in the logs.

Is this feature not implemented?

Attached should be the files I'm trying to convert.
[removed]

**Comment by Corak667** [comment]
Yes, currently it need to be converted to PSF2 with Vgmtools, it's easier way to get bank. 
I dont like it too and waiting developer to make normal workaround for bd+bh combo. Also it needs somehow to know what BD file made for BH. 

Program can load mass list of files (and still they need to be linked somehow), but still dont have any batch method to save multiple files at once... So it's another nonuserfrendly problem...

Still there are most rips only with PSF2 compiled, alternatively you can convert with fictional driver+sq to PSF2.
https://cloud.mail.ru/public/42NZ/oKkudALqV/Music_chip/PSF2_PlayStation_2/

**Comment by sykhro** [comment]
BD-BH linking shouldn't be a problem as long as the files follow the spec properly.
